src/script/api/ai_changelog.hpp

```diff
@@ -13,6 +13,14 @@
  * functions may still be available if you return an older API version
  * in GetAPIVersion() in info.nut.
  *
+ * \b Unknown
```

Comment:
no 1.10 freeze *yet*, this should be with the other 1.10 changes

Reply:
I meant it only as a temporary placeholder. :)